vernac/declare.ml

```diff
@@ -642,14 +642,31 @@ let declare_assumption ~name ~scope ~hook ~impargs ~uctx pe =
   dref
 
 (* Preparing proof entries *)
+let error_unresolved_evars env sigma t evars =
+  let pr_unresolved_evar e =
+    hov 2 (str"- " ++ Printer.pr_existential_key sigma e ++  str ": " ++
+      Himsg.explain_pretype_error env sigma
+        (Pretype_errors.UnsolvableImplicit (e,None))) in
```

Comment:
```suggestion
        (Pretype_errors.UnsolvableImplicit (e,None)))
  in
```
IMO this style is more readable, especially in the github diff view where lack of horizontal space can force line wrapping and so mess up the visual structure.